web_searchbar_full_width/static/src/xml/web_searchbar_full_width.xml

```diff
@@ -1,15 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates>
-    <t t-extend="ViewManagerAction">
-        <t t-jquery="tr.oe_header_row_top td:nth-child(1)">
-            jQuery(this).attr('colspan', 4);
-        </t>
-        <t t-jquery="tr.oe_header_row_top td:nth-child(2)">
-            var $new_row = jQuery('<tr class="oe_header_row"/>');
-            jQuery(this)
-            .parents('tr')
-            .after($new_row);
-            $new_row.append(jQuery(this).attr('colspan', 4));
+    <t t-extend="ControlPanel">
+        <t t-jquery="div.oe-cp-search-view" t-operation="replace">
```

Comment:
use no operation at all and write the class attribute via jquery as in the code you removed:
```xml
<t t-jquery="div.oe-cp-search-view">
    jQuery(this).removeClass('col-md-6');
    jQuery(this).addClass('col-md-12');
</t>
```
replacing nodes is never a good idea because you don't know which addons that possibly work on them are installed and what they do to the node.